lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.pm

```diff
@@ -0,0 +1,369 @@
+            is => "Text",
+            doc => "Output directory to write to",
+            is_output => 1,
+        },
+
+        gold_snv_vcf => {
+            is => "Text",
+            doc => "VCF file containing gold standard variant sites "
+                   . "to be used to measure the evaluation VCF (for snvs)"
+        },
+
+        gold_indel_vcf => {
+            is => "Text",
+            doc => "VCF file containing gold standard variant sites "
+                   . "to be used to measure the evaluation VCF (for indels)"
+        },
+
+        gold_sample => {
+            is => "Text",
+            doc => "Sample name in gold standard vcf file",
+            default => undef,
+        },
+
+        roi => {
+            is => "Text",
+            doc => "BED file of the target regions to restrict the analysis to",
+        },
+
+        true_negative_bed => {
+            is => "Text",
+            doc => "BED file containing regions where no "
+                   . "variant call should be made",
+        },
+
+        true_negative_size => {
+            is => "Integer",
+            doc => "Use this number as the size of the TN BED "
+                   . "file rather than calculating on the fly",
+            is_optional => '1',
+        },
+
+        pass_filter_expression => {
+            is => "Text",
+            doc => "String to pass to vcflib vcffilter in order to "
+                   . "select variants from the evaluation VCF",
+            default_value => q{-g 'FT = PASS | FT = .'},
+        },
+    ],
+
+    has_output => [
+        rawdata => {
+            is => "HASH",
+            doc => "The organized stats & metadata generated during "
+                   . "primary execution",
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+    my $configs = $self->parse_config_file();
+    $self->collect_statistics($configs);
+    $self->rawdata($configs);
+    $self->display_summary_stats();
+    return 1;
+}
+
+sub display_summary_stats {
+    my $self = shift;
+    my $configs = $self->rawdata;
+
+    $self->status_message("Generating Summary Report");
+
+    print "\n";
+    print "==================", "\n";
+    print "Valdiation Summary", "\n";
+    print "==================", "\n";
+    print "\n";
+
+    my @stat_names = $self->stat_types;
+    my @headers = ('Name', 'ID', 'VarType', @stat_names);
+    print join("\t", @headers), "\n";
+
+    for my $set (@{$configs}) {
+        my ($name, $id, $type, $stats) =
+          @{$set}{'name', 'id', 'variant_type', 'stats'};
+        my @results = @{$stats}{@stat_names};
+        print join("\t", $name, $id, $type, @results), "\n";
+    }
+}
+
+sub collect_statistics {
+    my ($self, $inputs) = @_;
+
+    my ($i, $total) = (1, scalar @{$inputs});
+    $self->status_message(
+        "Performing Evaluations & Collecting Statistics (%d vcfs)",
+        $total
+    );
+
+    for my $set (@{$inputs}) {
+        $self->status_message(
```

Comment:
Depending how noisy this will be, some of these might make more sense as `debug_message`s.